Lib/ufo2ft/postProcessor.py

```diff
@@ -78,6 +78,7 @@ def _rename_glyphs_from_ufo(self):
         # to TTX directly before compiling first, the post table will not
         # contain the extraNames.
         if "post" in otf and otf["post"].formatType == 2.0:
+            otf["post"].extraNames = []
```

Comment:
I wonder if this should be done instead in fonttools post table compile method?

Reply:
actually, no maybe better to do it here. If the post.compile method would always reset the extraNames, it would modify the order of existing post table and not roundtrip correctly (I think)

Reply:
Exactly, I considered doing this in FontTools, but was worried about this very reason.